<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QMainWindow" name="mainWindowClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Master's Thesis Project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="RenderingCanvas" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSome">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionUnload"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>Mesh</string>
    </property>
    <addaction name="actionWireframe"/>
    <addaction name="actionSubdivide"/>
    <addaction name="actionVertex_displacement"/>
    <addaction name="actionFace_displacement"/>
    <addaction name="actionExtract_displacements"/>
   </widget>
   <addaction name="menuSome"/>
   <addaction name="menuMesh"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load mesh</string>
   </property>
  </action>
  <action name="actionOFF">
   <property name="text">
    <string>OFF</string>
   </property>
  </action>
  <action name="actionOBJ">
   <property name="text">
    <string>OBJ</string>
   </property>
  </action>
  <action name="actionSubdivide">
   <property name="text">
    <string>Subdivision surfaces: Midpoint</string>
   </property>
   <property name="toolTip">
    <string>Subdivision surfaces: Midpoint</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionUnload">
   <property name="text">
    <string>Unload mesh</string>
   </property>
  </action>
  <action name="actionVertex_displacement">
   <property name="text">
    <string>Vertex displacement</string>
   </property>
  </action>
  <action name="actionFace_displacement">
   <property name="text">
    <string>Face displacement</string>
   </property>
  </action>
  <action name="actionExtract_displacements">
   <property name="text">
    <string>Extract displacements...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderingCanvas</class>
   <extends>QOpenGLWidget</extends>
   <header>renderingcanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>menuBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>menuBar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>mainToolBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
